include/ngp_utils/NgpLoopUtils.h

```diff
@@ -42,7 +42,11 @@ inline TeamPolicy
 ngp_mesh_team_policy(
   const size_t sz, const size_t bytes_per_team, const size_t bytes_per_thread)
 {
+#if defined(KOKKOS_ENABLE_HIP)
+  TeamPolicy policy(sz, 128);
```

Comment:
Can we make this number a macro so it only needs to be changed in one place going forward?

Reply:
> Can we make this number a macro so it only needs to be changed in one place going forward?

Good idea